src/test/java/uk/gov/hmcts/reform/prl/services/cafcass/CafcassCcdDataStoreServiceTest.java

```diff
@@ -0,0 +1,61 @@
+package uk.gov.hmcts.reform.prl.services.cafcass;
+
+import org.junit.Test;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.mockito.junit.MockitoJUnitRunner;
+import uk.gov.hmcts.reform.authorisation.generators.AuthTokenGenerator;
+import uk.gov.hmcts.reform.ccd.client.CoreCaseDataApi;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+import uk.gov.hmcts.reform.ccd.client.model.SearchResult;
+import uk.gov.hmcts.reform.prl.utils.TestResourceUtil;
+
+import java.io.IOException;
+import java.util.Arrays;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.mockito.Mockito.when;
+import static uk.gov.hmcts.reform.prl.utils.TestConstants.PRL_CASE_TYPE;
+import static uk.gov.hmcts.reform.prl.utils.TestConstants.TEST_AUTHORIZATION;
+
+@RunWith(MockitoJUnitRunner.class)
+public class CafcassCcdDataStoreServiceTest {
+
+    @Mock
+    private CoreCaseDataApi coreCaseDataApi;
+
+    @Mock
+    private AuthTokenGenerator authTokenGenerator;
```

Comment:
@Mock
    private AuthTokenGenerator authTokenGenerator; we need remove this , as we are not using

Reply:
Done